src/App.tsx

```diff
@@ -16,6 +16,7 @@ const App: FC = () => {
           <Auth />
         </Route>
         <Route path="*">
+          <Redirect to="/404-page" />
```

Comment:
name it not-found, please

Reply:
> name it not-found, please

<Route path="/not-found">
  <NotFound />
</Route>
<Route path="*">
  <Redirect to="/not-found" />
</Route>

Reply:
> > name it not-found, please

Reply:
> name it not-found, please

<Route path="/not-found">
  <NotFound />
</Route>
<Route path="*">
  <Redirect to="/not-found" />
</Route>